### ISSUE TITLE
Video size doesn't appear

### ISSUE THREAD

**Issue by Mohammed1520** [deniscerri/ytdlnis#278]
Video size doesn't appear on this picture. I use MP4 as default format. I use latest version of the app . Android 10
![Screenshot_20230912_144326_com deniscerri ytdl_edit_759846387959053](https://github.com/deniscerri/ytdlnis/assets/107951133/9d246ea8-6cd8-4000-b748-f8fd742ada9d)

**Comment by not-programmer98** [comment]
Could you reopen this? I still face this bug only with mp4 files

**Comment by not-programmer98** [comment]
It doesn't show the size of any mp4 format
![Imagepipe_0](https://github.com/deniscerri/ytdlnis/assets/126913926/b710c15d-22a9-4d07-b0a9-cae37928c780)

**Comment by deniscerri** [comment]
Its not related to mp4. Those formats have an unknown filesize even to yt-dlp

**Comment by not-programmer98** [comment]
But yt-dlp show a size with option -F. It's an almost size but better than nothing.
![ResizedImage_2023-09-25_06-13-47_8582](https://github.com/deniscerri/ytdlnis/assets/126913926/edac3a8e-73a8-48dc-aff5-04168dcb751a)

**Comment by deniscerri** [comment]
Yes but with -j it doesn't show it. The only way to get info is with j. That's how you can parse it.

Ytdlp would show it before but they removed it. We complained but they didn't want to change
